.github/workflows/arm-runner.yml

```diff
@@ -57,11 +57,11 @@ jobs:
             sudo rm -rf /pynab
             sudo apt-get autoremove -y
             sudo apt-get clean
-    - name: Compress the release image with pishrink
+    - name: Compress the release image with PiShrink
       run: |
         mv ${{ steps.arm_runner_install.outputs.image }} pynab-${{ steps.image_and_branch_name.outputs.IMAGE_NAME_SUFFIX }}.img
-        wget https://raw.githubusercontent.com/Drewsif/PiShrink/fa3c7a27406e7c0a1c7b8688e330f14a766be1ba/pishrink.sh
-        sudo bash pishrink.sh -Z -a pynab-${{ steps.image_and_branch_name.outputs.IMAGE_NAME_SUFFIX }}.img
+        wget https://raw.githubusercontent.com/Drewsif/PiShrink/master/pishrink.sh
+        sudo bash pishrink.sh -Z -a -p pynab-${{ steps.image_and_branch_name.outputs.IMAGE_NAME_SUFFIX }}.img
```

Comment:
We should not reintroduce -p IMHO. Deleting subfolders of /var/log is incoherent with what installed packages expect as the past issue with nginx proved. Even if we recreate what nginx does, we could encounter issues later with other packages.

Reply:
As requested, I moved back to not using the -p option.
But I slightly disagree with your argument: I have often cleaned up `/var/log` content, including subdirectories, and never had any problem with any package, **except** nginx.
We face here a genuine shortcoming/bug in the nginx package, which should be able to recreate the `/var/log/nginx `directory on startup if it needs it. For example, PostgreSQL has no such issue with the `/var/log/postgresql` directory.